.github/composite-actions/check-babelfish-inconsistency/action.yml

```diff
@@ -6,6 +6,6 @@ runs:
     - name: Check Babelfish metadata inconsistency
       id: check-babelfish-inconsistency
       run: |
-        output=$(sqlcmd -S localhost -U jdbc_user -P 12345678 -Q "SELECT sys.check_for_inconsistent_metadata() GO" | tail -n +2)
-        echo "check_result=$output" >> "$GITHUB_OUTPUT"
+        output=$(sqlcmd -S localhost -U jdbc_user -P 12345678 -Q "SELECT 'check_result=',sys.check_for_inconsistent_metadata() GO" | grep 'check_result' | sed 's/[[:blank:]]//g')
```

Comment:
I don't fully understand this line but I assume this converts the query result as follows:
```
check_result=  0         ---->   check_result=0
```
Is it the case?

Reply:
Yes, we are trimming the leading spaces here and solving the multi-line string issue that was occurring initially.